pkg/skaffold/deploy/kustomize.go

```diff
@@ -105,6 +107,8 @@ func NewKustomizeDeployer(cfg Config, labels map[string]string) *KustomizeDeploy
 		kubectl:            deploy.NewCLI(cfg, cfg.Pipeline().Deploy.KustomizeDeploy.Flags),
 		insecureRegistries: cfg.GetInsecureRegistries(),
 		globalConfig:       cfg.GlobalConfig(),
+		runID:              cfg.GetRunID(),
+		addRunIDAnnotation: cfg.AddSkaffoldLabels(),
```

Comment:
nit: The naming is a little bit confusing. AddXXX is expected to return an error used as "if nil, operation succeeds (true), otherwise gives the concrete error message"